ert_storage/graphql/ensembles.py

```diff
@@ -15,6 +15,28 @@ class Ensemble(SQLAlchemyObjectType):
     class Meta:
         model = ds.Ensemble
 
+    num_realizations = gr.Int()
+    child_ensembles = gr.List(lambda: Ensemble)
+    parent_ensemble = gr.Field(lambda: Ensemble)
+    responses = gr.Field("ert_storage.graphql.responses.Response")
+    parameters = gr.Field("ert_storage.graphql.parameters.Parameter")
+
+    def resolve_num_realizations(root: ds.Ensemble, info: "ResolveInfo") -> int:
+        return 5
```

Comment:
What's the reason for this?

Reply:
It's other stuff I need for webviz-ert. Will remove in this PR. :)